src/modules/assessments/components/household/HouseholdAssessments.tsx

```diff
@@ -257,6 +257,8 @@ const HouseholdAssessments: React.FC<Props> = ({
     px: 4,
   };
 
+  const showSummaryTab = enrollment.access.canEditEnrollments;
```

Comment:
I took the approach of simply hiding the whole summary tab, instead of making that tab view-only, since I don't think anything on there is relevant to a user who can view the enrollment but not edit any assessment data.